test/framework/task.go

```diff
@@ -14,14 +14,41 @@
 
 package framework
 
+import (
+	"fmt"
+	"strings"
+)
+
 // GetTasks retrieves tasks of specific process instance
 func GetTasks(namespace, routeURI, processName, processInstanceID string) (foundTasks map[string]string, err error) {
 	err = ExecuteHTTPRequestWithUnmarshalledResponse(namespace, "GET", routeURI, processName+"/"+processInstanceID+"/tasks", "", "", &foundTasks)
 	return
 }
 
+// GetTasksByUser retrieves tasks of specific process instance and user
+func GetTasksByUser(namespace, routeURI, processName, processInstanceID, user string) (foundTasks map[string]string, err error) {
+	err = ExecuteHTTPRequestWithUnmarshalledResponse(namespace, "GET", routeURI, processName+"/"+processInstanceID+"/tasks?user="+user, "", "", &foundTasks)
```

Comment:
Could you use `fmt` instead of concatenating strings? (same for all occurrences)

Reply:
Changed